csfieldguide/templates/interactives/mips-assembler.html

```diff
@@ -0,0 +1,185 @@
+      <h1>{% trans 'MIPS Assembler' %}</h1>
+    </div>
+    {% url 'interactives:interactive' 'mips-simulator' as simulator_link %}
+    <p>
+      {% blocktrans %}
+      Enter MIPS code below to see the assembler output.<br>
+      Then copy the plain output to the <a href="{{simulator_link}}" target="_blank">MIPS Simulator</a> interactive to run it.<br>
+      <strong>Note:</strong> Only a subset of MIPS is implemented. Comments start with #.<br>
+      {% endblocktrans %}
+    </p>
+    <details id="details">
+      <summary>{% trans 'Supported Instructions' %}</summary>
+      {% trans 'The following instructions are supported by both MIPS interactives' %}:<br>
+      <code>sll</code>, <code>jr</code>, <code>add</code>, <code>addu</code>, <code>sub</code>, <code>subu</code>, <code>and</code>, <code>or</code>, <code>xor</code>, <code>nor</code>, <code>beq</code>, <code>bne</code>, <code>addi</code>, <code>addiu</code>, <code>andi</code>, <code>ori</code>, <code>xori</code>, <code>j</code> <br>
+      {% blocktrans %}The <code>la</code>, <code>li</code> and <code>move</code> instructions are also supported when translated to appropriate instructions from the list.{% endblocktrans %}<br>
+      {% trans 'See all MIPS instructions on <a href="https://en.wikipedia.org/wiki/MIPS_architecture" target="_blank">Wikipedia</a>.' %}
+    </details>
+
+    <!-- Assembler -->
+    <div class="row justify-content-center">
+      <!-- Input Column -->
+      <div class="col-md-6 col-sm-12 mb-2">
+        <h3>{% trans 'MIPS Input' %}</h3>
+        <!-- Code Block -->
+        <div class="row textarea-padding">
+          <textarea id="mips-input" class="code">
+          </textarea>
+        </div>
+      </div>
+
+      <!-- Output Column -->
+      <div class="col-md-6 col-sm-12 mb-2">
+        <h3>{% trans 'Assembler Output' %}</h3>
+        <div id="assembler-output" class="code scroll">
+        </div>
+      </div>
+    </div>
+
+    <!-- Options Block -->
+    <div class="row justify-content-left">
+      <div class="col-lg-2">
+        <div class="form-group form-check">
+          <input id="show-blank" type="checkbox" class="form-check-input">
+          <label class="form-check-label" for="show-blank">{% trans 'Show context' %}</label>
+        </div>
+        <div class="form-group form-check">
+          <input id="show-instructions" type="checkbox" class="form-check-input">
+          <label class="form-check-label" for="show-instructions">{% trans 'Show details' %}</label>
+        </div>
+        <div class="form-group form-check">
+          <input id="show-colours" type="checkbox" class="form-check-input">
+          <label class="form-check-label" for="show-colours">{% trans 'Show colours' %}</label>
+        </div>
+      </div>
+      <div class="col-lg-5">
+        <button id="reset-adv" type="button" class="btn btn-outline-danger">{% trans 'Load advanced example' %}</button>
```

Comment:
Can we please add the mb-2 class to both of the 'load example ..' buttons?

Then it will go from looking like this on different screen sizes:
![screen shot 2019-02-15 at 11 41 20 am](https://user-images.githubusercontent.com/16066822/52822703-587b5f80-3117-11e9-88d0-9b481177a383.png)

to this:
![screen shot 2019-02-15 at 11 42 18 am](https://user-images.githubusercontent.com/16066822/52822709-5ca77d00-3117-11e9-8911-ab5574cf19f4.png)
![screen shot 2019-02-15 at 11 42 34 am](https://user-images.githubusercontent.com/16066822/52822715-5f09d700-3117-11e9-9ef1-50ddfaa23eea.png)